package.json

```diff
@@ -82,6 +82,7 @@
     "gl-spikes2d": "^1.0.1",
     "gl-surface3d": "^1.3.0",
     "mapbox-gl": "^0.22.0",
+    "matrix-camera-controller": "^2.1.3",
```

Comment:
`matrix-camera-controller` isn't required directly by plotly.js, but by including it as so in out package.json, we guarantee (in `npm@3` at least) that `npm i` will grab version `2.1.3` and up.